app/src/main/java/net/gsantner/markor/format/asciidoc/AsciidocActionButtons.java

```diff
@@ -29,8 +29,8 @@ public AsciidocActionButtons(@NonNull Context context, Document document) {
     }
 
     @Override
-    public List<ActionItem> getActiveActionList() {
```

Comment:
I think the diff is broken. Please rebase and re-create this PR

Reply:
It was indeed, showed 77 changed files now 3 changed files